src_python/habitat_sim/simulator.py

```diff
@@ -437,6 +438,13 @@ def get_sensor_observations(
             return next(iter(observations.values()))
         return observations
 
+    def debug_draw(self, sensor_uuid: Optional[str] = None) -> None:
```

Comment:
Shouldn't this be attached to the sensor object instead?

Reply:
I thought about this, but it adds extra complexity to the system overall. Users would need to override each sensor's debug_draw, possibly deriving the sensor type also (e.g. to only draw in one of multiple RGB sensors) instead of defining all the debug visualizations in one, centralized place.

Reply:
Yeah, but one of the longterm goals is to remove the sensor dict from the Simulator overall and this just complicates this down the road.

Reply:
fair point, I'll think on this a bit